crates/propolis-config-toml/src/spec.rs

```diff
@@ -71,7 +71,7 @@ pub enum TomlToSpecError {
 #[derive(Clone, Debug, Default)]
 pub struct SpecConfig {
     pub enable_pcie: bool,
-    pub components: BTreeMap<SpecKey, ComponentV0>,
+    pub components: HashMap<SpecKey, ComponentV0>,
```

Comment:
why change this to `HashMap`? not a blocker, just courious

Reply:
I think this must have leaked from an earlier revision of this change. (This *can* matter because `propolis_client::types::InstanceSpecV0`'s `components` member is a `HashMap`, whereas the `propolis_api_types` version of the type is a BTreeMap.) But changing this back doesn't seem to break anything in the current version of the change, so I'll flip it back just to reduce the number of lines this churns.